ThunderTable/TableViewController.swift

```diff
@@ -113,14 +113,19 @@ open class TableViewController: UITableViewController, UIContentSizeCategoryAdju
     
     open var data: [Section] {
         set {
+            // If the table view has had rows removed/added
```

Comment:
Might be worth documenting the reorder conundrum we have, just so it's somewhere :)

Reply:
See `rememberEmbeddedScrollPositions` :D

Reply:
Nice, thanks :)